solr/CHANGES.txt

```diff
@@ -220,6 +220,8 @@ Other Changes
 * SOLR-17347: EnvUtils: Removed "env" methods, as they were problematic.  Code should almost never
   refer to env vars.  (David Smiley)
 
+* SOLR-17322: Once again allow rank queries to use custom TopDocsCollectors that operate on types that extend ScoreDocs. (Stephen Woods via Christine Poerschke)
```

Comment:
"Once again" -- can we be more specific ... since what Solr version?

Reply:
I updated the CHANGES to include that Solr 9.0 broke it.